### ISSUE TITLE
convert: WavPack can be either lossless or lossy

### ISSUE THREAD

**Issue by tzhao11** [beetbox/beets#2789]
### Problem

Currently `convert` treat `wv` files as [lossy](https://github.com/beetbox/beets/blob/5f26ec6db9a852f623ea4dfeb2f99f7b43bb819b/beetsplug/convert.py#L46), so all WavPack files are copied instead of converted if `convert: never_convert_lossy_files` is set to `yes`, even when the `wv` files are actually lossless or hybrid. It seems [current logic](https://github.com/beetbox/beets/blob/5f26ec6db9a852f623ea4dfeb2f99f7b43bb819b/beetsplug/convert.py#L104) is not sufficient to determine losslessness of WavPack files correctly. 

According to http://www.wavpack.com/wavpack_doc.html#wvunpack, `wvunpack -f9` can be used to obtain the encoding mode of a `wv` file. Can we check [`MODE_LOSSLESS`](https://github.com/dbry/WavPack/blob/76eab1023836f53501c0e3bc5d1702e62a3330c5/include/wavpack.h#L322) and [`MODE_HYBRID`](https://github.com/dbry/WavPack/blob/76eab1023836f53501c0e3bc5d1702e62a3330c5/include/wavpack.h#L323) to handle `wv` files correctly?

**Comment by sampsyo** [comment]
Good point; thanks! We don’t currently have a way exposed inside beets to determine the WavPack mode, so we’d need to build that first and then use it in that convert setting.
